modules/installation/pages/install-vm.adoc

```diff
@@ -78,29 +78,65 @@ During the basic installation prompts you are asked to enter the root password.
 Select a strong password and then in the next message box btn:[Confirm root Password].
 ====
 
+
+
 [[jeos.susemgr.prep]]
 == Preparing JeOS for {productname}
 
 [[proc.jeos.susemgr.prep]]
 .Procedure: Preparing JeOS for {productname} Installation
-. Register with SCC:
+
+. Log in as `root`.
+
+// 4.0 workaround
+. Uninstall the `sles-release` package:
++
+
+----
+rpm -e --nodeps sles-release
+----
+
+. Register {productname} with SCC:
 +
 
 ----
-SUSEConnect -e<EMAIL_ADDRESS> -r<SUSE_MANAGER_CODE>
+SUSEConnect -e<EMAIL_ADDRESS> -r<SUSE_MANAGER_CODE> \
+  -p SUSE-Manager-Server/<productnumber>/x86_64
 ----
+
 . Add {productname} repositories:
 +
 
 ----
-SUSEConnect -p SUSE-Manager-Server/<productnumber>/x86_64 -r<SUSE_MANAGER_CODE>
+SUSEConnect -p sle-module-basesystem/15.1/x86_64
+SUSEConnect -p sle-module-python2/15.1/x86_64
+SUSEConnect -p sle-module-server-applications/15.1/x86_64
+SUSEConnect -p sle-module-web-scripting/15.1/x86_64
+SUSEConnect -p sle-module-suse-manager-server/<productnumber>/x86_64
+----
++
+TIP: JeOS is configured to install only Required packages. To get all
```

Comment:
Either remove the initcap, or mark up "required"

Reply:
Also needs a newline for the new sentence.

Reply:
ok, prepare to push